torch/__init__.py

```diff
@@ -50,28 +50,80 @@ def _running_with_deploy():
 
 
 __all__ = [
```

Comment:
ruff rule RUF022 will take care of this for us once it's out of preview.

Reply:
Not sure if we'd like to enable RUF022 globally.

Reply:
Why? Sorting the `__all__` list seems harmless.

Reply:
Sometimes, we add comments in `__all__` for better readability.

```python
__all__ = [
    # Module APIs
    "AaaModule",
    "BbbModule",
    "CccModule",
    # Dataset APIs
    "AaaDataset",
    "BbbDataset",
    "CccDataset",
]
```

After sorting:

```python
__all__ = [
    "AaaModule",
    "AaaDataset",
    "BbbModule",
    "BbbDataset",
    "CccModule",
    "CccDataset",
]
```